static/app/bootstrap/renderDom.tsx

```diff
@@ -13,6 +14,17 @@ export function renderDom(
     return;
   }
 
-  const root = createRoot(rootEl);
-  root.render(<Component {...props} />);
+  // Types are for ConfigStore, the window object is from json and features is not a Set
+  if (
+    (window.__initialData.features as unknown as string[]).includes(
+      'organizations:react-concurrent-renderer-enabled'
```

Comment:
organizations is not truncated here like other feature flags. see this other example https://github.com/getsentry/sentry/blob/483456bfea318484569667dadf8820b5e351e578/static/app/utils/withDomainRequired.tsx#L130-L132